internal/ask/form/submission/submission.go

```diff
@@ -138,6 +140,41 @@ func (s *Submission) Validate() error {
 	return nil
 }
 
+// GetQuestions retrieves the questions
+func (s *Submission) GetQuestions() []string {
+	var h []string
+
+	for _, r := range s.Answers {
+		h = append(h, r.Question.(string))
+	}
+
+	return h
+}
+
```

Comment:
I'd rather see the `GetQuestions()` and `GetAnswers()` methods as functions on the `submission` package to keep with our Function Oriented Design we've kept to so far. Especially as `GetQuestions` seems a bit confusing as it's returning a slice of strings for the CSV. Same applies to the `GetAnswers()` method.